1.12.0/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM nginx:alpine
+
+WORKDIR /usr/share/nginx/html
+
+ENV OPENHIM_CONSOLE_VERSION 1.12.0
+
+RUN wget "https://github.com/jembi/openhim-console/releases/download/v$OPENHIM_CONSOLE_VERSION/openhim-console-v$OPENHIM_CONSOLE_VERSION.tar.gz"
+
+RUN tar xzf "openhim-console-v$OPENHIM_CONSOLE_VERSION.tar.gz"
```

Comment:
Can we rather combine this `RUN` command with the one above to reduce the number of layers in our image? Something similar to https://github.com/jembi/openhim-docker/blob/f2fae87c96c26ef452b24ea99f4dd5d2a38b176a/1.12.0-rc.2/Dockerfile#L9. I think there's a way to do that with wget.

Reply:
aah ok cool, wasnt aware that `RUN` makes a new layer within the image :+1: